test.scss

```diff
@@ -0,0 +1,6 @@
+$color: #FFFFFF;
```

Comment:
Hex values should use the shorthand format - 3 characters where possible (hex-length)